t/016-resp-header.t

```diff
@@ -2141,3 +2141,63 @@ hi
 --- error_log
 my Content-Length: 8589934591
 upstream prematurely closed connection while sending to client
+
+
+
+=== TEST 95: Expose the 'Last-Modified' response header as ngx.header["Last-Modified"]
+--- config
+    location /a.txt {
+        header_filter_by_lua_block {
+            local last_modified = ngx.header["Last-Modified"]
+            if last_modified == nil then
+                ngx.log(ngx.ERR, "can not get lasted modified")
+                ngx.exit(500)
+                return
+            end
+
+
+            local last_mod = ngx.parse_http_time(last_modified)
+            local age = ngx.time() - last_mod
+            ngx.header["Age"] = age
+        }
+    }
+--- user_files
+>>> a.txt
+Foo
+--- request
+GET /a.txt
+--- raw_response_headers_like chomp
+Age: \d\r\n
+--- no_error_log
+[error]
+
+
+
+=== TEST 96: Expose the 'Last-Modified' response header as ngx.header["Last-Modified"]
```

Comment:
duplicated test name, better use a unique one.
Do you want to test the header is not existing?